widget/playground/src/components/ItemPicker/ItemPicker.styles.ts

```diff
@@ -1,15 +1,34 @@
 import { styled } from '@rango-dev/ui';
 
-export const TextContainer = styled('div', {
+export const InputContainer = styled('div', {
   border: '1px solid $neutral300',
   display: 'flex',
   justifyContent: 'space-between',
-  padding: '$10 $15',
+  padding: '$10',
   width: '100%',
-  borderRadius: '$sm',
+  borderRadius: '$xm',
   cursor: 'pointer',
   alignItems: 'center',
   '&:hover': {
     borderColor: '$info300',
   },
+  variants: {
+    disabled: {
+      true: {
+        cursor: 'default',
+        '&:hover': {
+          borderColor: '$neutral300',
```

Comment:
While hovering the input component, we need to change icon color as well based on Figma.

![Recording 2023-10-21 at 12 36 56](https://github.com/rango-exchange/rango-client/assets/99180934/7b9af95a-7a4f-42d8-864d-7823432e6301)

Reply:
fixed.